metabase/metabase
# Replace redux-form in Metric and Segment forms

Epic https://github.com/metabase/metabase/issues/22372

This is a completely custom form. This PR refactors it from `redux-form` to `formik`. 

How to test:
- Go to Admin -> Data Model -> Metrics
- Add a new metric with title, description, and filters
- Edit this metric
- Make sure that changes were saved correctly
- Do the same for Segments

<img width="1728" alt="Screenshot 2022-08-09 at 16 35 58" src="https://user-images.githubusercontent.com/8542534/183662431-77ff823b-5c12-443e-ab0e-4b32d77fdf51.png">
